Code/Framework/AzCore/AzCore/std/function/function_base.h

```diff
@@ -591,8 +591,8 @@ namespace AZStd
 
             Internal::function_util::function_buffer type_result;
             type_result.type.type = aztypeid(Functor);
-            type_result.type.const_qualified = is_const<Functor>::value;
-            type_result.type.volatile_qualified = is_volatile<Functor>::value;
+            type_result.type.const_qualified = std::is_const<Functor>::value;
+            type_result.type.volatile_qualified = std::is_volatile<Functor>::value;
```

Comment:
We have AZStd:: versions of these traits(There are just aliases to the std ones,).

Reply:
thanks, it was missing the header